File: Intermediate/Topic-003-Introduction-to-Arrays/Homework/Q1-Linear-Search-Multiple-Occurences/Linear-Search-Multiple-Occurences.py
class Solution:
    # @param A : list of integers
    # @param B : integer
    # @return an integer
    def solve(self, A, B):
        occur = 0
        for i in range(len(A)):
            if (B == A[i]):
                occur += 1
        
        return occur

File: Intermediate/Topic-003-Introduction-to-Arrays/Homework/Q1-Linear-Search-Multiple-Occurences/Linear-Search-Multiple-Occurences.c
/**
 * @input A : Integer array
 * @input n1 : Integer array's ( A ) length
 * @input B : Integer
 * 
 * @Output Integer
 */
int solve(int* A, int n1, int B) {

    int max, idx;

    max = 0;

    for (idx = 0; idx < n1; idx++) {
        if (B == A[idx]) {
            max += 1;
        }
    }
    return max;
}
